packages/preact/README.md

```diff
@@ -7,23 +7,23 @@ Signals is a performant state management library with two primary goals:
 
 Read the [announcement post](https://preactjs.com/blog/introducing-signals/) to learn more about which problems signals solves and how it came to be.
 
-## Installation:
-
-```sh
-npm install @preact/signals
-```
-
-- [Guide / API](../../README.md#guide--api)
```

Comment:
Renamed to 'Core API' as it's more clear, IMO. Not the complete 'Guide / API', just the core bit that the integrations builds upon. Anyone disagree?